controllers/datafoundation.go

```diff
@@ -158,6 +159,18 @@ func (r *dataFoundationReconciler) reconcilePhases() (ctrl.Result, error) {
 			r.Log.Info("Found OCS storage consumers, cannot proceed with uninstallation")
 			return ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
 		}
+
+		if err := r.get(&r.storageCluster); err != nil {
```

Comment:
- I still can't believe this was missed earlier, are you sure that there was no legit reason behind it?

Reply:
I was not able to think of a reason why it was not present at the first place